### ISSUE TITLE
Constructing a development environment.

### ISSUE THREAD

**Issue by ddyer0** [nolanlawson/KeepScore#88]
(Back from vacation, renewing my attempt to become a contributor)

To recap:  I can now build the original maven project and deploy it directly to my android device.  I'd like to upgrade this to a modern IDE where I can deploy and debug on a simulator from my windows desktop.  I'm trying to construct an eclipse based project.  And I'm down to two tantalizing errors.

Description	Resource	Path	Location	Type
dependency=[com.actionbarsherlock:actionbarsherlock:apklib:4.2.0:compile] not found in workspace	pom.xml	/keepscore	line 1	me.gladwell.eclipse.m2e.android.markers.dependency.apklib

error: Error retrieving parent for item: No resource found that matches the given name 'Widget.Sherlock.Light.ActionBar.TabText'.	styles.xml	/keepscore/res/values	line 16	Android AAPT Problem

---
Or as an alternative, tell me what  development environment I should use

**Comment by nolanlawson** [comment]
You have to import ActionBarSherlock into your Eclipse workspace.

And yeah, at the time I wrote this, it was Eclipse. I admit I haven't touched the codebase in years; it is probably pretty out of date. Welcome to Android, where your projects stop working after only a few years :stuck_out_tongue:

**Comment by ddyer0** [comment]
Yes, that's what I did to get down to the 2 remaining errors, that and for some unknown reason, the generated R.java is not made part of the project, so I added it manually.

**Comment by ddyer0** [comment]
Maybe If I simplify the question.  Here's one of the lines in eclipse that still issues a complaint.

   <style name="simpleHoloLightMenuItemTextView" parent="Widget.Sherlock.Light.ActionBar.TabText" />

the complaint is that "No resource found that matches the given name 'Widget.Sherlock.Light.ActionBar.TabText'"
.	styles.xml	/keepscore/res/values	line 16	Android AAPT Problem

Whereas, in the actionbarsherlock project, abs_styles.txt line 105, there is

<style name="Widget.Sherlock.Light.ActionBar.TabText" parent="Widget.Sherlock.ActionBar.TabText">
which seems to be a match.

Which leaves me, a novice at this mess, mystified.

**Comment by nolanlawson** [comment]
Are you using ABS version 4.2.0? It might depend on an older version than the current one.

Otherwise you could try importing a Maven project into Android Studio.

**Comment by nolanlawson** [comment]
Heh, I tried importing into Android Studio, and I get the same error as you. Guess this project is pretty messed up these days.

**Comment by ddyer0** [comment]
Some other messages point me at "minSdkVersion" and want me to increase it to at least 14.
I'm not sure what the implications of that would be, but perhaps the root of the problem is linking
this old codebase against a modern version of ActionBarSherlock.   What version was current when
keepscore was last released?

**Comment by nolanlawson** [comment]
I was using 4.2.0: https://github.com/nolanlawson/KeepScore/blob/ce6b2d839b405e6986e69e40defa80b4f81d9e1f/KeepScore/pom.xml#L18-L23

**Comment by ddyer0** [comment]
Hmm.  The other theory I'm entertaining is that the IDEs have become more clever at detecting potential conflicts, so the current errors are just newly detected; they were always there.  After all, the maven project still builds fine.

**Comment by sheindel** [comment]
@ddyer0 @nolanlawson did you ever have any luck with this? I'm sure digging into a now several year old codebase would be a nightmare but I'd love to contribute to keep this app alive.

**Comment by ddyer0** [comment]
I've moved on.  Rather than directly coding android apps, these days I use CodenameOne
